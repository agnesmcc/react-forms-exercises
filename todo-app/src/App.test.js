import { render } from '@testing-library/react';
import App from './App';

it('renders without crashing', () => {
  const { getByText } = render(<App />);
  const heading = getByText(/Todo List/);
  expect(heading).toBeInTheDocument();
})

it('matches the snapshot', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
}) 

