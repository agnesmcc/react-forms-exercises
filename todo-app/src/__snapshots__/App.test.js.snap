// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <h1>
        Todo List
      </h1>
      <div>
        <form>
          <div>
            <label
              for="task"
            >
              Task
            </label>
            <input
              id="task"
              name="task"
              type="text"
              value=""
            />
          </div>
          <button>
            Add a new todo
          </button>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
